db/.gitignore

```diff
@@ -0,0 +1 @@
+pgdata
```

Comment:
NIT: Unless there's a strong reason for having another gitignore, I'd prefer to use only one out of convention.